redigomock.go

```diff
@@ -18,14 +18,16 @@ type queueElement struct {
 // Conn is the struct that can be used where you inject the redigo.Conn on
 // your project
 type Conn struct {
-	ReceiveWait bool            // When set to true, Receive method will wait for a value in ReceiveNow channel to proceed, this is useful in a PubSub scenario
-	ReceiveNow  chan bool       // Used to lock Receive method to simulate a PubSub scenario
-	CloseMock   func() error    // Mock the redigo Close method
-	ErrMock     func() error    // Mock the redigo Err method
-	FlushMock   func() error    // Mock the redigo Flush method
-	commands    []*Cmd          // Slice that stores all registered commands for each connection
-	queue       []queueElement  // Slice that stores all queued commands for each connection
-	stats       map[cmdHash]int // Command calls counter
+	Subscription bool            // Enable this to support SubResponses
```

Comment:
Why do we need this flag? Isn't enough to have something in the SubResponses slice?